_layouts/default.html

```diff
@@ -8,7 +8,7 @@
     <link rel="stylesheet" type="text/css" media="screen" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
     <!-- Mathjax Support -->
     <script type="text/javascript" async
-            src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_CHTML">
+            src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?...">
```

Comment:
Are you sure of this change? I have few questions here:
- is it better to use cloudflare instead of the offficial cdn (probably yes)?
- what does latest means (snapshot or latest release)? I saw there is no equivalent in cloudflare.
- what does the `...` means? is it really necessary?

Reply:
- I found a warning while debugging the evolution and it gave me a link to [that webpage](https://www.mathjax.org/cdn-shutting-down/) where mathjax recommends changing CDN provider.
- "..." is an error, I modify it
- latest means the latest version. I can reuse the latest if prefered, or enforce version 2.7.1 as done in the modification, that prevents getting unwanted modification.

Reply:
Thanks for the explanations: we keep like this